obsolete/odata-aggregation-testcases.xml

```diff
@@ -1,491 +1,649 @@
-  </TestCase>
-  <TestCase Name="controlling aggregation - rollup and from" Rule="queryOptions">
-    <Input>$apply=groupby((rollup($all,Customer/Country,Customer/ID),Currency/Code),aggregate(Amount with sum from Customer/ID with average from Customer/Country with average as CustomerCountryAverage))</Input>
-  </TestCase>
-  <TestCase Name="transformation sequences - filter and aggregate" Rule="queryOptions">
-    <Input>$apply=filter(Amount le 1)/aggregate(Amount with sum as Total)</Input>
-  </TestCase>
-  <TestCase Name="transformation sequences - pre- and post-filter" Rule="queryOptions">
-    <Input>$apply=filter(Amount le 2)/groupby((Product/Name),aggregate(Forecast))&amp;$filter=Total ge 4</Input>
-  </TestCase>
-  <TestCase Name="transformation sequences - post-aggregate" Rule="queryOptions">
-    <Input>$apply=groupby((Time),aggregate(Amount with sum as Total))/aggregate(Total with average as DailyAverage)</Input>
-  </TestCase>
-  <TestCase Name="transformation sequences - simple" Rule="queryOptions">
-    <Input>$apply=groupby((Continent/Name,Country/Name),aggregate(Population with sum as TotalPopulation))</Input>
-  </TestCase>
-  <TestCase Name="transformation sequences - Population 1" Rule="queryOptions">
-    <Input>$apply=filter(Population ge 10000000)/groupby((Continent/Name,Country/Name),aggregate(Population with sum as TotalPopulation))</Input>
-  </TestCase>
-  <TestCase Name="transformation sequences - Population 2" Rule="queryOptions">
-    <Input>$apply=groupby((Continent/Name,Country/Name),aggregate(Population with sum as CountryPopulation))/filter(CountryPopulation ge 10000000)/concat(identity,groupby((Continent/Name),aggregate(CountryPopulation with sum as TotalPopulation)))</Input>
-  </TestCase>
-  <TestCase Name="transformation sequences - Population 3" Rule="queryOptions">
-    <Input>$apply=groupby((Continent/Name,Country/Name),aggregate(Population with sum as CountryPopulation))/filter(CountryPopulation ge 10000000)/groupby((rollup(Continent/Name,Country/Name)),aggregate(CountryPopulation with sum as TotalPopulation))</Input>
-  </TestCase>
-  <TestCase Name="transformation sequences - Population 4" Rule="queryOptions">
-    <Input>$apply=groupby((Continent/Name,Country/Name),aggregate(Population with sum as CountryPopulation))/concat(filter(CountryPopulation ge 10000000),groupby((Continent/Name),aggregate(CountryPopulation with sum as TotalPopulation)))</Input>
-  </TestCase>
-  <TestCase Name="transformation sequences - filter and expand" Rule="queryOptions">
-    <Input>$apply=filter(Status eq 'incomplete')/expand(Items,filter(not Shipped))/groupby((Customer/Country),aggregate(Items(Amount with sum as ItemAmount)))</Input>
-  </TestCase>
-  <TestCase Name="aggregate in $expand" Rule="odataRelativeUri">
-    <Input>Categories?$expand=Products($apply=aggregate(Price with average as AveragePrice))</Input>
-  </TestCase>
-</TestSuite>
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Are these space changes (like line ending)? ... I cannot see that with how I operate the GitHub browser diff UI, sorry.